addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -32,6 +32,26 @@ const {
 var qweb = core.qweb;
 var _t = core._t;
 
+// Patch ui.sortable to take into consideration the iframe of the wysiwyg.
+$.widget( "ui.sortable", $.ui.sortable, {
+    _generatePosition: function( event ) {
+        const pos = this._super(event);
+        if (this.options.wysiwygIframe) {
+            const offset = this.options.wysiwygIframe.offset();
+            pos.top -= offset.top;
+            pos.left -= offset.left;
+        }
+        return pos;
+    },
+    _mouseDistanceMet: function(...args) {
+        if (this.options.wysiwygIframe) {
+            return true;
+        } else {
+            return this._super(...args);
+        }
+    },
+});
+
```

Comment:
Could you please be more specific on which _feature_ of mass mailing does not work without this commit ? Because dropping snippets seems to work well. Where do we specifically need _sortable_ ?

Reply:
Commit message changed to:
```
[FIX] web_editor: patch jquery sortable to work with iframe

Before this commit, trying to drag&drop using jquery sortable
in mass mailing did not work.
The jquery sortable is used in the `ListUserValueWidget`.
The `ListUserValueWidget` is used in the `SocialMedia` snippet
option.
The reason was:
1. The iframe prevented the `window` main event to fire.
2. The position of the dragged element was not accurate.

task-2739302
```